<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>通讯录</string>
  </property>
  <widget class="MouseEvent" name="avatar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>个人主页</string>
   </property>
   <property name="accessibleName">
    <string>个人信息</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="Img.qrc">:/Img/Arous.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
   <property name="buddy">
    <cstring></cstring>
   </property>
  </widget>
  <widget class="QLabel" name="id">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>名称</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bonds">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>点券</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Img.qrc">:/Img/Bonds.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bonds">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>66</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>点券</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="member">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>会员等级</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Img.qrc">:/Img/VIP1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_updata">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>点击刷新数据</string>
   </property>
   <property name="text">
    <string>刷新数据</string>
   </property>
  </widget>
  <widget class="MouseEvent" name="label_updatasign">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>524</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>点击查看更新说明</string>
   </property>
   <property name="accessibleName">
    <string>更新日记</string>
   </property>
   <property name="text">
    <string> 更新说明 </string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_friends">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>371</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
  </widget>
  <widget class="MouseEvent" name="label_addfriends">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>添加好友</string>
   </property>
   <property name="accessibleName">
    <string>添加好友</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Img.qrc">:/Img/AddFriends.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_searchline">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="MouseEvent" name="label_search">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>搜索好友</string>
   </property>
   <property name="accessibleName">
    <string>搜索联系人</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Img.qrc">:/Img/Search.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>清空搜索</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Img/Delete.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
  </widget>
  <widget class="MouseEvent" name="label_shop">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>商城</string>
   </property>
   <property name="accessibleName">
    <string>商城</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Img.qrc">:/Img/Shop.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MouseEvent" name="label_membercenter">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>会员中心</string>
   </property>
   <property name="accessibleName">
    <string>会员中心</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Img.qrc">:/Img/Member.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MouseEvent" name="label_knapsack">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>背包</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleName">
    <string>背包</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Img.qrc">:/Img/Knapsack.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_retrun">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>527</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>点击刷新数据</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <zorder>avatar</zorder>
  <zorder>id</zorder>
  <zorder>bonds</zorder>
  <zorder>member</zorder>
  <zorder>btn_updata</zorder>
  <zorder>label_updatasign</zorder>
  <zorder>label_bonds</zorder>
  <zorder>listWidget_friends</zorder>
  <zorder>label_addfriends</zorder>
  <zorder>lineEdit_searchline</zorder>
  <zorder>label_search</zorder>
  <zorder>btn_delete</zorder>
  <zorder>label_shop</zorder>
  <zorder>label_membercenter</zorder>
  <zorder>label_knapsack</zorder>
  <zorder>btn_retrun</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MouseEvent</class>
   <extends>QLabel</extends>
   <header>mouseevent.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Img.qrc"/>
 </resources>
 <connections/>
</ui>
